pyrealm/pmodel/functions.py

```diff
@@ -13,7 +13,8 @@
 
 def calc_ftemp_arrh(
     tk: NDArray,
-    ha: float,
+    ha: float | NDArray,
+    tk_ref: NDArray | None = None,
```

Comment:
We are slightly inconsistent on whether or not we record optional args as optional in the doctoring description.

Reply:
Yeah - I agree. It's implicit in the typing that something is optional but I haven't been consistent about actual documenting it and saying what the defaults are. Reviewing this is probably a separate issue but I've fixed this one.